sys/include/net/cord/config.h

```diff
@@ -7,8 +7,8 @@
  */
 
 /**
- * @defgroup    net_cord_config CoRE RD Endpoint and Lookup Client Configuration
- * @ingroup     net_cord
+ * @defgroup    net_cord_config CoRE RD Configuration
+ * @ingroup     net_cord config
```

Comment:
```suggestion
 * @ingroup     net_cord
 * @ingroup     config
```

At least that's how it is done elsewhere.